src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements FiguresBehavior {
+    protected double square;
```

Comment:
it is better not to create this variable right away, because in the future you may change the value of the radius or the side and accordingly you will need to call the calculation of the area again. Let's just create a method that will calculate the area for us